docs/content/about/configuration.md

```diff
@@ -1155,6 +1170,53 @@ one of the `allow` regular expressions **and** one of the following holds:
 2. `deny` is set but no URLs within the manifest match any of the `deny` regular
    expressions.
 
+### `imageindexes`
+
+By default the registry will validate that all platform images exist when an image
+index is uploaded to the registry. Disabling this validatation is experimental
+because other tooling that uses the registry may expect the image index to be complete.
+
+validation:
+  imageindexes:
+    platformsexist: [All|None|Selected]
+    selectedplatforms:
```

Comment:
I feel like we are repeating ourselves here with `selected` prefix.

If the user sets `platformsexist` to `Selected` then the configured map implies it's selected, so we could just name the `selectedplatforms` to simply `platforms`.

I also feel like we could rename `Selected` to `Listed` and `platformexist` to simply `platforms`.

Maybe something like this could work better 🤔 (it does in my head 🙃):

```yaml
validation:
  index:
    platforms: [all|none|list]
    platformlist:
    - os: linux
       arch: amd64
```

Reply:
Yes, that's cleaner. I'm not sure how it ended up so convoluted in the first place :)